2/FDE/DomandeOrale/domande_elettronica.md

```diff
@@ -0,0 +1,55 @@
+*primo studente*
```

Comment:
I singoli file possono essere messi direttamente nella cartella della materia, solo nel caso in cui un singolo contenuto sia formato da diversi file serve farne una cartella

Reply:
Resta da cancellare questo file